src/viewProviders/resources.ts

```diff
@@ -88,7 +89,9 @@ type ResourceViewProviderData =
   | LocalResources
   | DirectResources;
 
-export class ResourceViewProvider implements vscode.TreeDataProvider<ResourceViewProviderData> {
+export class ResourceViewProvider
+  implements vscode.TreeDataProvider<ResourceViewProviderData>, RefreshableTreeViewProvider
+{
```

Comment:
Do we need this interface implementation set for [SchemasViewProvider](https://github.com/confluentinc/vscode/blob/53ab9242ed2b87c8ec11e0465e01aa72fbce639e/src/viewProviders/schemas.ts#L35) and [TopicViewProvider](https://github.com/confluentinc/vscode/blob/53ab9242ed2b87c8ec11e0465e01aa72fbce639e/src/viewProviders/topics.ts#L37) in this branch?

Reply:
They conform, proven at variable assignment line. Otherwise the assignment would be flagged as a typescript error.